wordguess.rb

```diff
@@ -0,0 +1,109 @@
+# Emilce C  & Maria Velez
+# Ada C9 - Octos - Word-Guess
+# 12th - 14th Feb 2018
+# https://github.com/Ada-C9/Word-Guess
+
+class Art
+  attr_accessor :art
+
+  def initialize
+    @art = art
+  end
+
+  def show_art
+     show_art = "(=^ ◡ ^=)"
+     return show_art
+  end
+
+  def art_array
+    art_array = show_art.split("")
+    puts "#{art_array}"
+    return art_array
+  end
+
+  def eliminate_art
+    show_art.slice(0)
+  end
+
+
+end
+
+class Word
+  attr_accessor :guess_letter
```

Comment:
This is another place where the name of the class, `Word`, indicates it will be responsible for some small piece of functionality, but instead you've ended up throwing the whole game in here. You should either rename the class to something like `Game`, or move functionality out.

Knowing what your classes and methods are _not_ responsible for is just as important as knowing what they _are_ supposed to do.